pajbot/web/utils.py

```diff
@@ -68,8 +68,7 @@ def no_cache(*args, **kwargs):
 
 
 def download_logo(twitch_helix_api, streamer):
-    streamer_id = twitch_helix_api.require_user_id(streamer)
-    logo_url = twitch_helix_api.get_profile_image_url(streamer_id)
+    logo_url = twitch_helix_api.get_profile_image_url(StreamHelper.get_streamer_id())
```

Comment:
I think we should either pass in both the streamers name and id, or fetch both from StreamHelper, I think I prefer the first option